app/src/pages/inside/filtersPage/filtersPage.jsx

```diff
@@ -29,12 +32,22 @@ import { FILTERS_PAGE, FILTERS_PAGE_EVENTS } from 'components/main/analytics/eve
 import { NoFiltersBlock } from './noFiltersBlock';
 import { FilterPageToolbar } from './filterPageToolbar';
 import { FilterGrid } from './filterGrid';
+import styles from './filtersPage.scss';
 
+const cx = classNames.bind(styles);
 const messages = defineMessages({
   filtersPageTitle: {
     id: 'FiltersPage.title',
     defaultMessage: 'Filters',
   },
+  filtersNotFound: {
```

Comment:
Duplicated messages. See the comment below

Reply:
I do not understand this comment. This is `filtersNotFound` and the other is `membersNotFound`

Reply:
Oh, sorry. I meant this message (checkQuery)[https://github.com/reportportal/service-ui/pull/1129/files#diff-340ef0ec1098ee0c60d2688b5fcf9173R47]

Reply:
fixed